ssl/statem/extensions_clnt.c

```diff
@@ -108,8 +108,8 @@ static int use_ecc(SSL *s)
     return i < end;
 }
 
-int tls_construct_ctos_ec_pt_formats(SSL *s, WPACKET *pkt, X509 *x,
-                                     size_t chainidx, int *al)
+int tls_construct_ctos_ec_pt_formats(SSL *s, WPACKET *pkt, unsigned int context,
```

Comment:
Hmmm, I'm unconvinced. That would just introduce unnecessary complexity IMO, i.e. code to pack the struct prior to the call. It has no purpose other than to pass parameters to a function...which is what function parameters are for! I don't really see 6 parameters as a problem that needs solving.